documentation/guidelines.md

```diff
@@ -0,0 +1,6 @@
+# Colors
+
```

Comment:
Please specify that this is a guideline to designing UI elements for oot

Reply:
makes sense